### PR TITLE
Change back image of cards to react logo

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

File: public/img/5.svg [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-  <path class="cls-2" d="M56.74 43.36a8.87 8.87 0 1 1-8.86 8.87 8.87 8.87 0 0 1 8.86-8.87" transform="translate(-.76 -2.14)"/>
-<head xmlns=""/><style xmlns="" type="text/css" id="igtranslator-color"/></svg>
\ No newline at end of file
+<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 267.69 303.2">
```
Comment (probil) [inline public/img/5.svg]: Consider using some of those resources for SVG optimization:
https://squoosh.app
https://jakearchibald.github.io/svgomg/

File: public/img/react.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 111.74 100.46">
```
Comment (probil) [inline public/img/react.svg]: Consider using some of those resources for SVG optimization:
https://squoosh.app
https://jakearchibald.github.io/svgomg/

File: src/Card/Card.js [hunk 3]
```diff
@@ -13,8 +13,8 @@ const Card = (props) => {
           visibility: props.cardProperties.hidden ? "hidden" : "visible",
         }}
       >
-        <img src=  "img/js-badge.svg" alt = {props.cardProperties.id} className = "backImg"/>
-        <img src={`img/${props.cardProperties.hiddenValue}.svg`} alt = {props.cardProperties.id} className = "frontImg"/>
+        <img src= "img/react.svg" alt = {props.cardProperties.id} className = "backImg"/>
+        <img src= {`img/${props.cardProperties.hiddenValue}.svg`} alt = {props.cardProperties.id} className = "frontImg"/>
```
Comment (probil) [inline src/Card/Card.js]: I think changes in this line are useless

File: public/img/5.svg (hunk 1 above)
Comment (liubich) [reply public/img/5.svg]: > https://squoosh.app

This stuff may convert to another (raster) format only. I think, this isn't a good solution, is it?

**Comment by liubich** [comment]
@probil, thank you for review, but this PR wasn't ready for it. I want to change background and timer text colors for better view.

File: public/img/5.svg (hunk 1 above)
Comment (probil) [reply public/img/5.svg]: Yeah https://squoosh.app is good png/jpg/webp. For SVG use https://jakearchibald.github.io/svgomg/

**Pull request merged**